src/main/java/com/softserve/academy/spaced/repetition/DTO/impl/CourseTopDTO.java

```diff
@@ -15,6 +15,10 @@ public CourseTopDTO(Course course, Link link) {
         add(linkTo(methodOn(DeckController.class).getAllDecksByCourseId(getEntity().getCategory().getId(), getEntity().getId())).withRel("decks"));
     }
 
+    public Long getCourseId(){return getEntity().getId();}
```

Comment:
What happened to formatting?

Reply:
Because on web I need to show rating and allow authorized user to change it.

Reply:
No problem, but please reformat the code, make it look nice.

Reply:
Oh, sorry, I didn't understand it at first.